src/security/kuzzleProfile.js

```diff
@@ -1,7 +1,7 @@
 var
   KuzzleSecurityDocument = require('./kuzzleSecurityDocument');
```

Comment:
This should also apply to this variable name. And this modification should be applied to all subsequent files.